cloud_storage/cloud_storage/overrides/file.py

```diff
@@ -42,10 +42,33 @@ def validate(self) -> None:
 		if self.flags.cloud_storage or self.flags.ignore_file_validate:
 			return
 		if not self.is_remote_file:
-			super().validate()
+			self.custom_validate()
 		else:
 			self.validate_file_url()
 
+	def custom_validate(self):
```

Comment:
I think we should enable the "Track Overrides" feature on this one, since it overrides Frappe's file validate function. If the GHA is not added, I think we should add it to this repo.

Reply:
Yes, sounds like a good plan, Your thoughts @agritheory?